contracts/modules/assessment/IndividualClaims.sol

```diff
@@ -315,7 +315,7 @@ contract IndividualClaims is IIndividualClaims, MasterAwareV2 {
       (
         bool refunded,
         /* bytes data */
-      ) = msg.sender.call{value: msg.value - assessmentDepositInETH}("");
+      ) = tx.origin.call{value: msg.value - assessmentDepositInETH}("");
```

Comment:
what's the context for this change?

Completely changes semantics of returning the deposit

Reply:
As discussed offband this change in not necessary or appropriate.

We should use msg.sender and proxy the ET back and forth to the original msg.sender.

For all intents and purposes IndividualClaims.submitClaim should only refer to msg.sender.